src/meta/proto-conv/src/schema_from_to_protobuf_impl.rs

```diff
@@ -226,6 +226,7 @@ impl FromToProto for ex::TableDataType {
                 let x = n.to_pb()?;
                 new_pb_dt24(Dt24::NumberT(x))
             }
+            TableDataType::Decimal(_) => todo!("decimal"),
```

Comment:
`unimplemented!  `  or `unreachable` is better.

Old version dt do not have this path.